timescaledb/how-to-guides/page-index/page-index.js

```diff
@@ -174,6 +174,13 @@ module.exports = [
             keywords: ["hypertables", "hyperfunctions", "TimescaleDB"],
             excerpt: "Use advanced analytics queries",
           },
+          {
+            title: "Faster queries with SkipScan",
```

Comment:
I think the title is more around: **"DISTINCT queries on time-series data"** than mentioning SkipScan in the title (again, few folks will know unless they read out blog)

Any thoughts to add @svenklemm?

Reply:
I agree SkipScan is an internal name and most SQL user will probably not have heard it or be able to relate it to any SQL query.

Reply:
I just checked on Algolia, and we've had at least a few users search for "skipscan" or "skip scan" so I don't want to remove it entirely.